activesupport/lib/active_support/concern.rb

```diff
@@ -104,6 +104,12 @@ def initialize
       end
     end
 
+    class UnreachableIncludedBlock < StandardError #:nodoc:
+      def initialize
+        super "Cannot add an 'included' block after Concern was already included"
```

Comment:
If I explicitly add require to both it seems they are both loaded twice, and then the multiple included block errors gets thrown:

```
app/models/example.rb:6: warning: already initialized constant Example::A
app/models/example.rb:6: warning: previous definition of A was here
app/indices/example_concern.rb:7: warning: already initialized constant ExampleConcern::B
app/indices/example_concern.rb:7: warning: previous definition of B was here
/Users/david/.rbenv/versions/2.1.6/lib/ruby/gems/2.1.0/gems/activesupport-4.2.4/lib/active_support/concern.rb:126:in `included': Cannot define multiple 'included' blocks for a Concern (ActiveSupport::Concern::MultipleIncludedBlocks)
        from app/indices/example_concern.rb:9:in `<module:ExampleConcern>'
```